tests/Helper/PhpMqttClientBuilder.php

```diff
@@ -0,0 +1,41 @@
+<?php declare(strict_types=1);
+
+
+namespace Lib\Tests\Helper {
+
+    use PhpMqtt\Client\Exceptions\ProtocolNotSupportedException;
+
+    /**
+     * Manages the PhpMqttClient with Logging.
+     * @package Lib\Tests\Helper
+     */
+    class PhpMqttClientBuilder
+    {
+        private MqttClient $mqttClient;
+
+        /**
+         * Constructor.
+         * @param string $host
+         * @param string $port
+         * @param string $clientId
+         * @throws ProtocolNotSupportedException
+         */
+        public function __construct(string $host, string $port, string $clientId)
```

Comment:
Not a builder pattern >>> move parameters from the constructor to dedicated fluent methods.